### PR TITLE
New sample file testing IfcLinearPlacement and related concepts

### PR DESCRIPTION
**Pull request by seb-esser**
# New Sample for IfcLinearPlacement

Fixes: likely nothing but might raise attention to issues around the linear positioning concept

## Changes and improvements proposed by this pull request
- LinPlacement sample 2

fyi:
@jmirtsch @pjanck @NigelPMPeters @SergejMuhic @larswik 

# IFC4.3 Unit test

## Unit test
not decided yet - file could help developers to get a solid starting point when implementing alignment and linear referencing methods in recent IFC versions

## Review requested
- IfcAlignment and its axis should be ok
- IfcBuildingElementProxy instances should be ok
- FZK Viewer reports an issue on the `IfcShapeRepresentation`: 
grafik
Any feedback and suggestions for improvements are highly appreciated

### CODE REVIEW

**Comment by larswik** [comment]
There are a number of IFCPRESENTATIONLAYERASSIGNMENT that references entities that doesn't exist. I also get some geometric errors (not telling me much at this stage). However, your issue on shape representation, I dont get in my version (6.0 - build 1658) of FZKViewer. Otherwise, it looks fine to me!
image

**Review by larswik — COMMENTED** [review]
Why change the FILE_SCHEMA to 'IFC4X3RC1' without the underscore? Otherwise fine by me.

File: IFC 4.3/LinearPlacement_2/LinearPlacement_2.ifc [hunk 1]
```diff
@@ -0,0 +1,252 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION ((''), '2;1');
+FILE_NAME ('', '2020-02-02T10:55:51', (''), (''), 'Processor version 0.0.0.0', 'Brain', '');
+FILE_SCHEMA (('IFC4X3RC1'));
```
Comment (pjanck) [inline IFC 4.3/LinearPlacement_2/LinearPlacement_2.ifc:5]:
```suggestion
FILE_SCHEMA (('IFC4X3_RC1'));
```

File: IFC 4.3/LinearPlacement_2/LinearPlacement_2.ifc [hunk 2]
```diff
@@ -0,0 +1,252 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION ((''), '2;1');
```
Comment (pjanck) [inline IFC 4.3/LinearPlacement_2/LinearPlacement_2.ifc:3]:
```suggestion
FILE_DESCRIPTION (('ViewDefinition []'), '2;1');
```

**Review by pjanck — CHANGES_REQUESTED** [review]
Minor changes in the header. Otherwise LFTM.

**Comment by larswik** [comment]
I would love to merge this pull request. However, I don't seem to have the permissions...
image

**Pull request merged**